src/unittests/floating_point.spec.ts

```diff
@@ -4903,12 +4903,15 @@ g.test('stepInterval')
           { input: [1, -1], expected: 0 },
 
           // 64-bit normals
-          { input: [0.1, 0.1], expected: [0, 1] },
+          // number is f64 internally, so the value for 0.1/-0.1 will be precise
```

Comment:
This is ok... but slightly misleading perhaps.  By "precise" did you mean "perfectly accurate"? Because  0.1 is not exactly representable in binary floating point. (Because 1/5 is not a terminating binary fraction).

use something like 0.375  ( = 3/8) for a perfectly representable fraction.

Reply:
I agree I wasn't clear here, I was getting at the fact that number is internally a f64, the same as abstract float, so no matter what normal literal we put in there, we will never be able to create the the step could go either way behaviour that lower precision floating points have. I will clarify the comment.